src/main/groovy/de/dkfz/roddy/tools/TimeUnit.groovy

```diff
@@ -151,4 +151,24 @@ class TimeUnit {
     String toString() {
         return timeString
     }
+
+    String alignToHours() {
```

Comment:
What do you mean by align? What is the method doing? Some math on times, o.k., but for what purpose?